frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -1,18 +1,148 @@
-import React from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GridContainer, Icon, SiteAlert } from "@trussworks/react-uswds";
 
 import { AuthElement } from "../AuthElement";
 import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { USLink } from "../USLink";
+import Spinner from "../Spinner";
+import { showError } from "../AlertNotifications";
 import { MemberType } from "../../hooks/UseOktaMemberships";
+import {
+    validateFileType,
+    ManagePublicKeyAction,
+    CONTENT_TYPE,
+    FORMAT,
+    validateFileSize,
+} from "../../hooks/network/ManagePublicKey/ManagePublicKeyHooks";
+
+import { ManagePublicKeyChooseSender } from "./ManagePublicKeyChooseSender";
+import { ManagePublicKeyUpload } from "./ManagePublicKeyUpload";
+
+const LightbulbIcon = Icon.Lightbulb;
+
+const ManagePublicKeySwitchDisplay = () => {
```

Comment:
Minor point, but I don't see much of a benefit of having ManagePublicKeySwitchDisplay being broken out of the ManagePublicKey component.  I feel like it can all be inside ManagePublicKey 🤷